### DISCUSSION TITLE
Courseplay für Konsole?

### DISCUSSION THREAD

**Discussion by Othi38** [Courseplay/Courseplay_FS22#1928] [Q&A] [answered]
Hallo zusammen,
meine Frage ist, ob courseplay auch für Konsolen möglich ist, bzw. ob es schon Pläne dafür gibt?

Gruß
Othi38

**Comment by Tensuko** [comment, marked as answer]
Diese Frage wurde 10000 mal gestellt und es gibt dazu nur eine Antwort:
Script Mods laufen nicht auf Konsole.
Diese Antwort wirst du überall im Internet finden.
